scripts/generate_genotype_df/generate_genotype_df.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+"""Generate genotype dfs for the association analysis"""
+
+import hail as hl
+from analysis_runner import bucket_path, output_path
+
+TOB_WGS = bucket_path('mt/v7.mt/')
+
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(TOB_WGS)
+    # filter out variants that didn't pass the VQSR filter
+    mt = mt.filter_rows(hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0)
+
+    # VQSR does not filter out low quality genotypes. Filter these out
+    mt = mt.filter_entries(mt.GQ <= 20, keep = False)
+    # filter out samples with a genotype call rate > 0.8 (as in the gnomAD supplementary paper)
+    n_samples = mt.count_cols()
+    call_rate = 0.8
+    mt = mt.filter_rows(hl.agg.sum(hl.is_missing(mt.GT)) > (n_samples * call_rate), keep=False)
+    # filter out variants with MAF < 0.01
+    mt = mt.filter_rows(mt.freq.AF[1] > 0.01)
+    # export as PLINK file
+    tob_wgs_path = output_path('tob_wgs_maf01.parquet')
+    mt.rows().to_spark().write.mode("append").parquet(tob_wgs_path)
```

Comment:
This will write a Spark DataFrame, not a Pandas DataFrame. Are you sure that's what you need downstream?

Reply:
This saves just fine as a parquet file that I can load in as a pandas df with `pd.read_parquet`. Is there a better way to do this?

Reply:
There's `to_pandas`, maybe that will have a slightly different format? @vladsaveliev, do you know?

Reply:
ok, I tested this out and while both dfs are the same when reading back into python, the `to_spark` way results in a much larger file. I've now changed this to `to_pandas` (and only take the fields I need), and then export to parquet

Reply:
Cool, thanks! You probably also don't need the "append" write mode here, right?

Reply:
No, I just did the `mt.rows().to_pandas().to_parquet()`

Reply:
Perfect! 😸